js/src/forum/addUserInfo.js

```diff
@@ -16,7 +16,7 @@ export default function () {
     const findMatchClass = function (node, className) {
         const arr = [];
 
-        if (node && node.children) {
+        if (node && node.children && typeof node.children === 'object') {
```

Comment:
🤔 Might be better to do `Array.isArray`, as objects aren't necessarily arrays.

Reply:
Done!